zone/embparser.cpp

```diff
@@ -1709,6 +1711,23 @@ void PerlembParser::ExportEventVariables(
 			break;
 		}
 
+		case EVENT_SKILL_UP: {
+			Seperator sep(data);
+			ExportVar(package_name.c_str(), "skill_id", sep.arg[0]);
+			ExportVar(package_name.c_str(), "skill_value", sep.arg[1]);
+			ExportVar(package_name.c_str(), "skill_max", sep.arg[2]);
+			ExportVar(package_name.c_str(), "is_tradeskill", sep.arg[3]);
```

Comment:
Don’t need is tradeskill flag anymore. Can tell that by skill ID.

Reply:
Agreed